docs/UserGuide.md

```diff
@@ -66,7 +66,7 @@ description`, with a deadline of `8 Sep 2020, 1700 hrs` and with an `urgent` tag
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Notes about the command format:**<br>
```

Comment:
might be fine to keep it as : : around the information_source 
this makes an emoji